lumigator/python/mzai/schemas/lumigator_schemas/jobs.py

```diff
@@ -88,6 +88,14 @@ class JobInferenceCreate(BaseModel):
     config_template: str | None = None
 
 
+class JobAnnotateCreate(BaseModel):
+    name: str
+    description: str = ""
+    dataset: UUID
+    max_samples: int = -1  # set to all samples by default
+    task: str | None = "summarization"
```

Comment:
do we pass the JobAnnotateCreate into the generic job response?

Reply:
We pass all it's contents and add the task-dependant model on top, to complete the JobInferenceCreate config. 
Currently we just have the one task, so there isn't much choice, but soon enough we'll want to distinguish between annotating for a translation job and a summarization job.